Pipfile

```diff
@@ -6,18 +6,16 @@ verify_ssl = true
 [dev-packages]
 pytest = "*"
 pytest-testinfra = "*"
-
-[packages]
-ansible = "==9.6.1"
-ansible-lint = "*"
-cryptography = "*"
 docker = "*"
 molecule = "*"
 molecule-docker = "*"
 pylint = "*"
 
-# Remove once this is fixed: https://github.com/ansible-collections/community.docker/issues/868#issuecomment-2124979976
-requests = "==2.31.0"
+[packages]
+ansible = "*"
+ansible-lint = "*"
+cryptography = "*"
+requests = "*"
```

Comment:
@dkinzer Did you verify that the issue related to this upgrade has been fixed?

Reply:
@sensei100 according to this the issue is resolved: https://github.com/ansible-collections/community.docker/issues/868#issuecomment-2129847836